File: Testing/Code for Testing/Early Basic Syntax Testing/dowhile/test_dowhile.py
from dataclasses import dataclass, field

def _ul_sort(arr, n): arr[:n] = sorted(arr[:n])

class _ul_Exception(Exception):
    def __init__(self, type_name, message):
        self.type_name = type_name
        self.message = message
        super().__init__(message)


def main():
    count = 0
    while True:
        print(count)
        count += 1
        if not ((count < 5)):
            break
    x = 100
    while True:
        print(x)
        x += 1
        if not ((x < 0)):
            break
    return 0


if __name__ == "__main__":
    main()

File: Testing/Code for Testing/Early Basic Syntax Testing/dowhile/test_dowhile.js

function main() {
    let count = 0;
    do {
        console.log(count);
        count += 1;
    } while ((count < 5));
    let x = 100;
    do {
        console.log(x);
        x += 1;
    } while ((x < 0));
    return 0;
}

function __ul_sort(arr, n) { const sub = arr.slice(0, n).sort((a, b) => a < b ? -1 : a > b ? 1 : 0); for (let i = 0; i < n; i++) arr[i] = sub[i]; }

main();
